app/.env.example

```diff
@@ -6,4 +6,5 @@ PGHOST=
 PGPORT=
 DATABASE_URL=
 ORIGIN="http://localhost:3000"
+PORT=3000
```

Comment:
Adding a configurable port means we need to make the port configurable in our dockerfile. Let's add a tech debt card for this.